src/app/store/api.js

```diff
@@ -3,10 +3,12 @@ import { createClient } from 'contentful-management'
 let client = null
 let space = null
 
-export async function initClient (accessToken, spaceId) {
+export async function initClient (accessToken, spaceId, host, uploadHost) {
   try {
     client = createClient({
-      accessToken
+      accessToken,
+      defaultHostname: host,
+      defaultHostnameUpload: uploadHost
```

Comment:
for a simple test we don't need to set ports here so setting `defaultHostname` should be enough